test/api/services/EnrolmentsAuthServiceSpec.scala

```diff
@@ -193,7 +193,7 @@ class EnrolmentsAuthServiceSpec extends ServiceSpec with MockAppConfig {
 
         MockAppConfig.confidenceLevelCheckEnabled.returns(ConfidenceLevelConfig(definitionEnabled = true, authValidationEnabled = false))
 
-        val expected = Left(StandardDownstreamError)
+        val expected: Left[StandardDownstreamError.type, Nothing] = Left(StandardDownstreamError)
```

Comment:
Please add "MtdError" as the error type of all the errors in EnrolmentsAuthServiceSpec and MtdIdLookupServiceSpec.